content/kotlin/concepts/datetime/terms/toDatePeriod/toDatePeriod.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'toDatePeriod()'
+Description: 'Parses the ISO-8601 duration representation as a DatePeriod.'
+Subjects:
+  - 'Date and Time'
+  - 'Computer Science'
+Tags:
+ - 'DatePeriod'
+ - 'DateTimePeriod'
+ - 'ISO-8601'
```

Comment:
```suggestion
 - 'Methods'
 - 'Data Structures'
 - 'Time'
```
Check valid [tags](https://github.com/Codecademy/docs/edit/main/documentation/tags.md) before adding them in meta data.